### ISSUE TITLE
Duplicate import errors when using Yara-X

### ISSUE THREAD

**Issue by FractalArt** [YARAHQ/yara-forge#43]
I am not sure if [`yara-x`](https://github.com/VirusTotal/yara-x) is supposed to be supported in the future but at the moment the `yara-forge` rule package files seem to cause failures when used with `yara-x` due to identical imports being used several times across the rule files.

I had a quick look [here](https://github.com/FractalArt/yara-forge/blob/c0c6f833952b06369a3da40ad4c1dcf59e01befc/main/rule_output.py#L267) and it seems like this could be fixed rather easily. I have no experience in contributions but if this is something that could be of use, I could open a pull request.

**Comment by Neo23x0** [comment]
Yes, the fix should be straight forward. I just didn't care at that time in late 2023.

**Comment by Neo23x0** [comment]
I'm working on it in https://github.com/YARAHQ/yara-forge/pull/52
